Library/ENV/4.3/cc

```diff
@@ -1,6 +1,10 @@
-#!/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby -W0
+#!/bin/sh
+# Make sure this shim uses the same Ruby interpreter that is used by Homebrew.
+unset RUBYLIB
+unset RUBYOPT
+exec "$HOMEBREW_RUBY_PATH" -x "$0" "$@"
```

Comment:
Maybe use same message in [`config.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/config.rb#L31) for scm wrapper?